FEATURES.md

```diff
@@ -0,0 +1,98 @@
+# Features
+
```

Comment:
I decided not to add table of contents because RDoc generates header anchors in a weird format, like `label-Document+Symbol` for `Document Symbol`, instead of just `document-symbol`.
Since `snapper` already has a table of contents on the side, it should be fine.